scripts/core/EXP-NATIVE-ENQUEUE.rst

```diff
@@ -32,13 +32,13 @@ within the native API through the function passed to
 ${x}EnqueueNativeCommandExp, the function argument must only use the native
 queue accessed through ${x}QueueGetNativeHandle. Use of a native queue that is
 not the native queue returned by ${x}QueueGetNativeHandle results in undefined
-behaviour.
+behavior.
 
-Any args that are needed by the func must be passed through a void* and unpacked
-within the func. If ${x}_mem_handle_t arguments are to be used within
-pfnNativeEnqueue, they must be accessed using ${x}MemGetNativeHandle.
-${x}_mem_handle_t arguments must be packed in the void* argument that will be
-used in pfnNativeEnqueue, as well as ${x}EnqueueNativeCommandExp's phMemList
+Any args that are needed by the func must be passed through a `void*` and unpacked
```

Comment:
I think single backticks is italics, double backticks for monospace

Reply:
putting `${x}` in inline monospace doesn't render well though

Reply:
it seems like it's working ok in this instance https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/EXP-MULTI-DEVICE-COMPILE.rst?plain=1#L55 https://oneapi-src.github.io/unified-runtime/core/EXP-MULTI-DEVICE-COMPILE.html#support